blog: health-checking-grpc

ahmetb opened the pull request:
date in the markdown and file/dir names will be updated later

neolit123 commented on content/en/blog/_posts/2018-09-30-health-checking-grpc.md:
```diff
@@ -0,0 +1,97 @@
+    numbers).
+2.  **tcpSocket probe:** Opening a socket to gRPC server is not meaningful,
+    since it cannot read the response body.
+3.  **exec probe:** This invokes a program in container's ecosystem
```
`in container's` -> `in a container's` ?

neolit123 commented on content/en/blog/_posts/2018-09-30-health-checking-grpc.md:
```diff
@@ -0,0 +1,97 @@
+2.  **tcpSocket probe:** Opening a socket to gRPC server is not meaningful,
+    since it cannot read the response body.
+3.  **exec probe:** This invokes a program in container's ecosystem
+    periodically. In gRPC case, this means you implement a health RPC yourself,
```
`In gRPC case` -> `In the case of gRPC` ?

neolit123 commented on content/en/blog/_posts/2018-09-30-health-checking-grpc.md:
```diff
@@ -0,0 +1,97 @@
+To standardize the "exec probe" approach mentioned above, we need:
+
+- a **standard** health check "protocol" that can be implemented in any gRPC
+  server easily,
```
`easily,` -> `easily.`

ahmetb commented:
Thanks for the review. Addressed feedback: [[staging url]](https://deploy-preview-10393--kubernetes-io-master-staging.netlify.com/blog/2018/09/30/health-checking-grpc/)

kbarnard10 commented:
/retest

kbarnard10 commented:
/lgtm
/approve
